### PR TITLE
PFCF-476: Changes to form 40

### PR DESCRIPTION
**Pull request by JonTaylorBCGov2**

### CODE REVIEW

File: web/src/components/steps/noticeLawyerChild/reviewNLC/pdf/Form40Layout.vue [hunk 1]
```diff
@@ -3,239 +3,235 @@
...
-            <i>Select all options that apply</i>
-            <div class="marginleft2p5vue" style="margin:0.25rem 0 0 1.5rem;">
-                <check-box 
-                    class="marginleft" 
-                    checkbox="" 
-                    inline="inline" 
-                    boxMargin="0" 
-                    style="display:inline;"
-                    :check="listOfIssues.includes('Parenting Arrangements')?'yes':''" text="parenting arrangements"/>
+            <div style="width: 35%; float:right; font-size: 8pt;">
```
Comment (HemanthKona) [inline web/src/components/steps/noticeLawyerChild/reviewNLC/pdf/Form40Layout.vue:16]: @JonTaylorBCGov2 There is FormHeader common component. May be better to use that to keep it consistent.

Comment (JonTaylorBCGov2) [reply web/src/components/steps/noticeLawyerChild/reviewNLC/pdf/Form40Layout.vue:16]: I tried using that, but it wouldn't produce the header - I was not sure what I was doing incorrectly, so I abandoned it in favour of something that worked for me. I'd be happy to change it, but I may need some assistance in getting it to work.

Comment (JTraill) [reply web/src/components/steps/noticeLawyerChild/reviewNLC/pdf/Form40Layout.vue:16]: Are you talking about 'family-law-act-app\web\src\components\utils\PopulateForms\components\FormHeader.vue' ?

I can get it to work, but Its not generic enough. Im currently making it more generic right now so my PR can use it
https://github.com/bcgov/family-law-act-app/pull/697

Comment (JTraill) [reply web/src/components/steps/noticeLawyerChild/reviewNLC/pdf/Form40Layout.vue:16]: Disregard, I wasnt working with the latest version. Im not observing the same as Jon

**Review by HemanthKona — APPROVED** [review]

**Pull request merged**